src/ModuleGroup.cc

```diff
@@ -4,14 +4,15 @@
 
 #include "ApplicationModule.h"
 #include "DeviceModule.h"
+#include "Utilities.h"
 
 namespace ChimeraTK {
 
   /********************************************************************************************************************/
 
   ModuleGroup::ModuleGroup(ModuleGroup* owner, const std::string& name, const std::string& description,
       const std::unordered_set<std::string>& tags)
-  : Module(owner, name, description, tags) {
+  : Module(owner, ChimeraTK::Utilities::stripTrailingSlashes(name), description, tags) {
```

Comment:
Would it be possible to move this stripping operation into the `Module` base class, or is it too late for some purposes? It would simplify the change, since all module types would benefit from a single change...